product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

```diff
@@ -72,7 +72,7 @@ An Advanced Server URL contains the following information:
 
 -   `{TARGET_DB_USER|SRC_DB_USER}` — Specifies a user with privileges to create each type of object migrated. If migrating data into a table, the specified user might also require insert, truncate, and references privileges for each target table.
 
--   `{TARGET_DB_PASSWORD|SRC_DB_PASSWORD}` — Set to the password of the privileged Advanced Server user.
+-   `{TARGET_DB_PASSWORD|SRC_DB_PASSWORD}` — Set to the password of the privileged Advanced Server user. Do not wrap the password in quotes.
```

Comment:
This and the following edits refer to sections in the doc that mention setting the password in the scope of defining a database URL depending on the provider (PG, BigAnimal, Oracle, SQL, etc.).  

For some reason, even though the password is not mentioned in that URL, setting a password is required in all "Defining URL" section of this page. 

1. Do we really need to mention the password in the "Defining URL" sections, even though they are not implicitly included in the URL? 
2. If yes, I am assuming the "do not use quotes" rule applies here as well? 

@zahid75 @mlewandowski-edb

Reply:
I think having the section titles saying "Defining ... URL", which is about the connection URL and then appending the migration toolkit SRC_DB_USER/TARGET_DB_USER and SRC_DB_PASSWORD/TARGET_DB_PASSWORD to the URL parameters discussion is very confusing. Would it make sense to retitled the sections to something like "Defining the URL and configuring other properties for <DB type connections>" and then more clearly separate the USER and PASSWORD property descriptions from the URL property descriptions.

Reply:
Sounds good, thanks for your replies. To sum up, on each of the "Defining ... URL" sections, I'll: 

- Change the title from "Defining ... URL" to "Defining the URL and configuring the required properties for ..."
- Separate the SRC_DB_USER/TARGET_DB_USER and SRC_DB_PASSWORD/TARGET_DB_PASSWORD bullet points from the other bullets with an introductory clause like "Ensure these properties are set in the `toolkit.properties` file:"